packages/compose/src/hooks/use-copy-on-click/index.js

```diff
@@ -48,7 +56,7 @@ export default function useCopyOnClick( ref, text, timeout = 4000 ) {
 
 			// Handle ClipboardJS focus bug, see https://github.com/zenorocha/clipboard.js/issues/680
 			if ( trigger ) {
-				trigger.focus();
+				/** @type {HTMLElement} */ ( trigger ).focus();
```

Comment:
I know this has been merged already, but just curious: why did you wrap `trigger` in parenthesis here?

Reply:
It's how you do a type cast of a variable in JSDoc flavored TypeScript: https://github.com/Microsoft/TypeScript/issues/5158